### ISSUE TITLE
save_returning() should also work when creating records

### ISSUE THREAD

**Issue by rvinzent** [M1hacka/django-pg-returning#20]
### Feature Request

Calling `save_returning()` only seems to work when the save performs an update. When the save performs an insert the database generated value is not returned.

works:
```python
    instance = MyModel.objects.create(
        name="test",
        description="a test with a database generated barcode",
        barcode="something fake",
    )
    instance.barcode = Concat(Value("TE"), Value("123"))
    instance.save_returning()
```

does not work:
```python
    instance = MyModel(
        name="test",
        description="a test with a database generated barcode",
        barcode=Concat(Value("TE"), Value("123")),
    )
    instance.save_returning()
```

A useful extension of this might be to add `create_returning()` method analogous to `update_returning()` except performing an `INSERT`.

**Comment by M1ha-Shvn** [comment]
This feature may be a tricky one, as it needs deep invasion into Django `Model.save()` internals. `save_returning` is implemented over django private methods which can be changed any time. 
I'm not sure, I'll have time to dig inside that thing in near time. Though, you can make a PR, if you are interested in it.

**Comment by M1ha-Shvn** [comment]
Released [v1.3.0](https://github.com/M1ha-Shvn/django-pg-returning/releases/tag/v1.3.0)
